### PR TITLE
Add documentation to housing selection [#159809098]

### PR DESCRIPTION
**Pull request by ajzoell**
Only things being added to this PR will be documentation adjustments on additions to this repository over the weekend.

### CODE REVIEW

File: src/Housing.Selection.Library/ServiceHubModels/ApiUser.cs [hunk 1]
```diff
@@ -3,14 +3,24 @@
...
+    /// <summary>
+    /// Service hub user model.
+    /// </summary>
     public class ApiUser
     {
+        /// <value>Service hub primary key.</value>
         public Guid UserId { get; set; }
+        /// <value>Location user is assigned to.</value>
         public string Location { get; set; }
+        ///<value>Service hub address where user is from.</value>
```
Comment (bjkirk94) [inline src/Housing.Selection.Library/ServiceHubModels/ApiUser.cs]: Confirmed with Nick this is the user's permanent home address.  (independent of what room they are placed into.)

File: src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs [hunk 2]
```diff
@@ -30,22 +26,34 @@ public ServiceBatchMessaging(IQueueFactory queueFactory)
...
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Get date and time of last service-hub modification.
+        /// </summary>
+        /// <returns>Date and time of the last service-hub modification.</returns>
         public Task<DateTime> GetLastModified()
         {
             //TODO: this
```
Comment (fredbelotte) [inline src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs]: Code Smell Code Smell: Complete the task associated to this 'TODO' comment. ([csharpsquid:S1135](https://sonarcloud.io/organizations/revaturecloud/rules#rule_key=csharpsquid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=housing%3Aselection&pullRequest=40&issues=AWV74S1AXKENQn1Sxgme&open=AWV74S1AXKENQn1Sxgme)

[ISSUE_KEY]: AWV74S1AXKENQn1Sxgme

File: src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs [hunk 3]
```diff
@@ -30,22 +26,34 @@ public ServiceBatchMessaging(IQueueFactory queueFactory)
...
+        /// Get batch current user is assigned to.
+        /// </summary>
+        /// <param name="userId">Id of user in corresponding service-hub database we want to look for.</param>
+        /// <returns>
+        /// Service-hub batch information for corresponding user. Currently, 
+        /// only throws NotImplementedException.
+        /// </returns>
         public Task<ApiBatch> GetBatchForUserAsync(Guid userId)
         {
             //TODO: this
```
Comment (fredbelotte) [inline src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs]: Code Smell Code Smell: Complete the task associated to this 'TODO' comment. ([csharpsquid:S1135](https://sonarcloud.io/organizations/revaturecloud/rules#rule_key=csharpsquid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=housing%3Aselection&pullRequest=40&issues=AWV74S1AXKENQn1Sxgmd&open=AWV74S1AXKENQn1Sxgmd)

[ISSUE_KEY]: AWV74S1AXKENQn1Sxgmd

File: src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs (hunk 2 above)
Comment (fredbelotte) [inline src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs]: Code Smell Code Smell: Complete the task associated to this 'TODO' comment. ([csharpsquid:S1135](https://sonarcloud.io/organizations/revaturecloud/rules#rule_key=csharpsquid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=housing%3Aselection&pullRequest=40&issues=AWV74S1AXKENQn1Sxgme&open=AWV74S1AXKENQn1Sxgme)

[ISSUE_KEY]: AWV74S1AXKENQn1Sxgme

File: src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs (hunk 3 above)
Comment (fredbelotte) [inline src/Housing.Selection.Context/ServiceBus/ServiceBatchMessaging.cs]: Code Smell Code Smell: Complete the task associated to this 'TODO' comment. ([csharpsquid:S1135](https://sonarcloud.io/organizations/revaturecloud/rules#rule_key=csharpsquid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=housing%3Aselection&pullRequest=40&issues=AWV74S1AXKENQn1Sxgmd&open=AWV74S1AXKENQn1Sxgmd)

[ISSUE_KEY]: AWV74S1AXKENQn1Sxgmd

**Comment by ajzoell** [comment]
Most recent commit is auto-generated

**Comment by ajzoell** [comment]
Also, bugs and code smells not addressed as I only added comments, not altered source code.

**Pull request merged**